frontend/src/pages/Web/index.js

```diff
@@ -31,13 +28,6 @@ const WebsiteTitle = styled('h3')({
 });
 
 class Web extends Component {
-  handleWebsiteClick = websiteId => {
```

Comment:
What's the reason for removing this? Without this function it breaks the `/web` page.
```
TypeError: this.handleWebsiteClick is not a function
```

Reply:
There was no reason at all. That shouldn't had been removed. My bad. (maybe some sort of multi cursor or while rebasing I lost it).